pkg/artifactory/resource/security/resource_artifactory_permission_target.go

```diff
@@ -361,13 +365,13 @@ func resourcePermissionTargetUpdate(ctx context.Context, d *schema.ResourceData,
 }
 
 func resourcePermissionTargetDelete(_ context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
-	_, err := m.(*resty.Client).R().Delete(permissionsEndPoint + d.Id())
+	_, err := m.(*resty.Client).R().Delete(PermissionsEndPoint + d.Id())
 
 	return diag.FromErr(err)
 }
 
 func PermTargetExists(id string, m interface{}) (bool, error) {
-	resp, err := m.(*resty.Client).R().Head(permissionsEndPoint + id)
+	resp, err := m.(*resty.Client).R().Head(PermissionsEndPoint + id)
```

Comment:
@alexhung is this still used anywhere? The new schema context did away with `exists` and now presumably only uses `ReadFunc`. `HEAD` is definitely really lightweight, but not every resource supported it well

Reply:
This is for the `testPermissionTargetCheckDestroy()` in the test so still needed.